test/xcode-specs.js

```diff
@@ -70,7 +81,7 @@ describe('xcode @skip-linux', () => {
     let path = await xcode.getAutomationTraceTemplatePath();
 
     should.exist(path);
-    fileExists(path).should.eventually.be.true;
+    fs.exists(path).should.eventually.be.true;
```

Comment:
Except `eventually` works on a promise. It could be `(await fs.exists(path)).should.be.true;` but otherwise this works.